tensorbay/client/requests.py

```diff
@@ -47,12 +47,7 @@
 
 
 class Config:  # pylint: disable=too-few-public-methods
-    """This is a base class defining the concept of Request Config.
-
-    Attributes:
-         is_intern: Whether the request is from intern.
-
-    """
+    """This is a base class defining the concept of Request Config."""
```

Comment:
`is_intern` should be renamed to `is_internal`, also should be change to a public attr